source/examples/rapids-sagemaker-higgs/rapids-higgs.py

```diff
@@ -4,8 +4,8 @@
 
 import cudf
 from cuml import RandomForestClassifier as cuRF
-from cuml.preprocessing.model_selection import train_test_split
-from sklearn.metrics import accuracy_score
+from cuml.metrics import accuracy_score
```

Comment:
This job currently fails like this:

```text
```text
 File "/opt/ml/code/rapids-higgs.py", line 67, in <module>
    main(args)
  File "/opt/ml/code/rapids-higgs.py", line 47, in main
    print("test_acc:", accuracy_score(cu_rf.predict(X_test), y_test))
  File "/opt/conda/lib/python3.10/site-packages/sklearn/utils/_param_validation.py", line 213, in wrapper
    return func(*args, **kwargs)
  File "/opt/conda/lib/python3.10/site-packages/sklearn/metrics/_classification.py", line 222, in accuracy_score
    y_type, y_true, y_pred = _check_targets(y_true, y_pred)
  File "/opt/conda/lib/python3.10/site-packages/sklearn/metrics/_classification.py", line 100, in _check_targets
    type_true = type_of_target(y_true, input_name="y_true")
  File "/opt/conda/lib/python3.10/site-packages/sklearn/utils/multiclass.py", line 316, in type_of_target
    if is_multilabel(y):
  File "/opt/conda/lib/python3.10/site-packages/sklearn/utils/multiclass.py", line 171, in is_multilabel
    y = check_array(y, dtype=None, **check_y_kwargs)
  File "/opt/conda/lib/python3.10/site-packages/sklearn/utils/validation.py", line 1007, in check_array
    array = _asarray_with_order(array, order=order, dtype=dtype, xp=xp)
  File "/opt/conda/lib/python3.10/site-packages/sklearn/utils/_array_api.py", line 746, in _asarray_with_order
    array = numpy.asarray(array, order=order, dtype=dtype)
  File "/opt/conda/lib/python3.10/site-packages/nvtx/nvtx.py", line 116, in inner
    result = func(*args, **kwargs)
  File "/opt/conda/lib/python3.10/site-packages/cudf/core/frame.py", line 408, in __array__
    raise TypeError(
TypeError: Implicit conversion to a host NumPy array via __array__ is not allowed, To explicitly construct a GPU matrix, consider using .to_cupy()
To explicitly construct a host matrix, consider using .to_numpy()
```

On this line:

https://github.com/rapidsai/deployment/blob/8bd5b05bdcfd23032daf7ee83724681416d6341d/source/examples/rapids-sagemaker-higgs/rapids-higgs.py#L37

Passing `cudf.Series` objects through `sklearn.accuracy_score()` isn't supported, at least in the combination of versions present in the `rapidsai/base:24.06a` images.

Switching from `sklearn.metrics.accuracy_score` to `cuml.metrics.accuracy_score` fixes that (and is probably what we'd rather recommend anyway).